### DISCUSSION TITLE
is there any guide, to integrate Node.js with ADFS?

### DISCUSSION THREAD

**Discussion by AnasHussain47** [bergie/passport-saml#824] [Q&A]
I have React Node js complete application and I have to integrate with ADFS for SSO anyone could help it will be appreciated ?

**Comment by markstos** [comment]
There are some docs in this repo. They are not actively maintained. They may be outdated and may later be deleted from this repo, as we don't have volunteers to maintain docs for all the different SSO providers. 

https://github.com/node-saml/passport-saml/blob/aef40fff6909dd8f23e3b7f09746a5e96243b1f4/docs/adfs/README.md

It would be better to maintain something like this on a wiki or a third-party site.
